BUILD.gn

```diff
@@ -79,47 +79,165 @@ if (is_fuchsia) {
       "$flutter_root/content_handler:aot",
     ]
     if (flutter_runtime_mode != "release") {
-      deps += [ "//third_party/dart/runtime/observatory:embedded_archive_observatory" ]
+      deps += [
+        "//third_party/dart/runtime/observatory:embedded_archive_observatory",
+      ]
+    }
+
+    binary = "flutter_aot_runner"
+
+    if (flutter_runtime_mode != "release") {
+      resources = [
+        {
+          path = rebase_path(
+                  "$root_gen_dir/observatory/embedded_archive_observatory.tar")
+          dest = "observatory.tar"
+        },
+      ]
+    }
+
+    meta = [
+      {
+        path = rebase_path("content_handler/meta/sandbox")
+        dest = "sandbox"
+      },
+    ]
+  }
+
+  package("flutter2_aot_runner") {
```

Comment:
This is an exact copy of flutter_aot_runner. It's purpose is to ensure Dart 2 and Dart 1 applications end up in separate processes since some of the Dart 2 flags are scoped to the whole process.